<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_selectCSVHeaders</class>
 <widget class="QMainWindow" name="MainWindow_selectCSVHeaders">
  <property name="windowTitle">
   <string>Text Columns?</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <layout class="QGridLayout" name="gridLayout_selectCSVHeaders_ActionButton">
        <item row="1" column="2" alignment="Qt::AlignBottom">
         <widget class="QPushButton" name="pushButton_selectCSVHeaders_save">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_selectCSVHeaders_checkAll">
          <property name="text">
           <string>check all</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_selectCSVHeaders_uncheckAll">
          <property name="text">
           <string>uncheck all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_selectCSVHeaders_editor"/>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>303</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
